de.fu_berlin.inf.dpp.whiteboard/META-INF/MANIFEST.MF

```diff
@@ -14,5 +14,6 @@ Require-Bundle: org.eclipse.ui,
  de.fu_berlin.inf.dpp
 Bundle-ActivationPolicy: lazy
 Bundle-ClassPath: .,
- lib/log4j-1.2.16.jar
+ lib/log4j-1.2.16.jar,
```

Comment:
I know this is hard to setup, but the actual used log4j is provided by Saros Eclipse.  So I do not know if the logging will still work as the log4j property file might not be found.

Reply:
this change has nothing to do with the logging file, but if I am not wrong I remember @m273d15 working on changing the version of log4j, this might be addressed in another patch.